EventsExpress/ClientApp/src/components/event/filter/parts/organizer/organizer-autocomplete.js

```diff
@@ -0,0 +1,60 @@
+import { Autocomplete } from '@material-ui/lab';
+import { InputAdornment, TextField } from '@material-ui/core';
+import React from 'react';
+import { useOrganizerFilterStyles } from './organizer-filter-styles';
+import { connect } from 'react-redux';
+import { getFormValues } from 'redux-form';
+import { fetchUsers } from '../../../../../actions/events/filter/users-data';
+import { useDelay } from './use-delay';
+
+export const OrganizerAutocomplete = ({ dispatch, input, options, formValues }) => {
```

Comment:
If this is a custom component - you don't get its values from the form-values. You need to get its values from the input.value. Please read documentation on custom inputs of react-redux-form.

Reply:
Done